### DISCUSSION TITLE
New Features

### DISCUSSION THREAD

**Discussion by motazKhogali** [MohmmedAshraf/laravel-translations#108] [Ideas]
Merge the In Translation project into the main Laravel project as a sidebar tab. Ensure it shares the user database and explore customization options.

**Comment by MohmmedAshraf** [comment]
Hi @motazKhogali i didn't get what you mean?

**Comment by motazKhogali** [comment]
- Integrate the laravel-translations into Laravel's views, 
 - enabling its use as a component or subview. 
 - Utilize Laravel's authentication functionalities instead of creating custom contributors,
 -  and explore options for additional customisation
